insights/tests/client/apps/test_playbook_verifier.py

```diff
@@ -339,23 +341,36 @@ def test_dict_list(self):
         assert result == expected
 
     def test_dict_mixed(self):
-        source = {"key": "key", "value": ["value1", "value2"]}
+        source = collections.OrderedDict([("key", "key"), ("value", ["value1", "value2"])])
         result = playbook_verifier.PlaybookSerializer._dict(source)
         expected = "ordereddict([('key', 'key'), ('value', ['value1', 'value2'])])"
         assert result == expected
 
     def test_dict_multiple(self):
-        source = {"key": "key", "value": "value"}
+        source = collections.OrderedDict([("key", "key"), ("value", "value")])
         result = playbook_verifier.PlaybookSerializer._dict(source)
         expected = "ordereddict([('key', 'key'), ('value', 'value')])"
         assert result == expected
 
     def test_numbers(self):
-        source = {"integer": 37, "float": 17.93233901}
+        source = collections.OrderedDict([("integer", 37), ("float", 17.93233901)])
         result = playbook_verifier.PlaybookSerializer._dict(source)
         expected = "ordereddict([('integer', 37), ('float', 17.93233901)])"
         assert result == expected
 
+    @pytest.mark.parametrize(
+        "source,expected",
+        [
+            ("no quote", "'no quote'"),
+            ("single'quote", '''"single'quote"'''),
+            ("double\"quote", """'double"quote'"""),
+            ("both\"'quotes", r"""'both"\'quotes'""")
+        ]
+    )
+    def test_strings(self, source, expected):
+        result = playbook_verifier.PlaybookSerializer._str(source)
```

Comment:
A bit nitpick, however it would be better to use `PlaybookSerializer.serialize()` here (and the above original tests) instead of the internal "._xxx" methods.